vocabularies/PDF.xml

```diff
@@ -74,6 +74,12 @@
             <String>If this option is selected, the font size is automatically selected in such a way that all columns of a table fit on one page. Other layout options like margin, border and composite cell spacing are adapted accordingly, with respect to the chose scaling factor.</String>
           </Annotation>
         </Property>
+        <Property Name="Padding" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
```

Comment:
```suggestion
        <Property Name="Padding" Type="Edm.Decimal" Nullable="false" DefaultValue="0">
```
If the value must be > 0, this cannot be a boolean value.

Reply:
It must be a boolean.

We can have a separate Frontend server. The annotation is to let the client know, if the server supports this option. This is the same alike FontName, FontSize and so on.

The actual type of the Padding is for sure an integer. But this is described in the separated OData service ($metadata) where you can set the format options like fontName, size and also padding